lib/releases.js

```diff
@@ -0,0 +1,110 @@
+var path = require('path');
+var _ = require('lodash');
+
+var Shipit = module.exports;
+
+/**
+ * Compute the current release dir name.
+ *
+ * @param {object} result
+ * @returns {string}
+ */
+
+var computeReleases = function computeReleases(result) {
+  if (!result.stdout) return null;
+
+  // Trim last breakline.
+  var dirs = result.stdout.replace(/\n$/, '');
+
+  // Convert releases to an array.
+  return dirs.split('\n');
+};
+
+/**
+ * Test if all values are equal.
+ *
+ * @param {*[]} values
+ * @returns {boolean}
+ */
+
+var equalValues = function equalValues(values) {
+  return values.every(function (value) {
+    return _.isEqual(value, values[0]);
+  });
+};
+
+/**
+ * Compute the current release dir name.
+ *
+ * @param {object} result
+ * @returns {string}
+ */
+
+var computeReleaseDirname = function computeReleaseDirname(result) {
```

Comment:
No problem, I'll change.
At Fusionary, we tend to always use function expressions to [clarify hoisting and declaration order](https://javascriptweblog.wordpress.com/2010/07/06/function-declarations-vs-function-expressions/). Just a preference.